File: acm_statistic_backend/api/api/DTO/performance/compagnie/RepartitionGlobalCompagnieMoisDTO.cs
namespace api.DTO.performance.compagnie
{
    public class RepartitionGlobalCompagnieMoisDTO
    {
        public int Mois { get; set; }
        public string Mois_Lettre { get; set; }
        public decimal? Montant { get; set; }
        public string Compagnie { get; set; }
        public int Vols { get; set; }
        public decimal? Part {  get; set; }
        public decimal? Pourcentage_variation { get; set; }

    }
}

File: acm_statistic_frontend/Interface Tableau de Bord Analytique Aérien/src/models/performanceCompagnie/RepartitionGlobalCompagnieMoisDTO.ts
export class RepartitionGlobalCompagnieMoisDTO{
    mois :number; 
    mois_lettre : string;
    vols :number; 
    montant?: number;
    compagnie?: string ;
    part? :number ; 
    pourcentage_variation? : number ;

    constructor(init?: Partial<RepartitionGlobalCompagnieMoisDTO>) {
    Object.assign(this, init);
    }   
}